optimus/wdl_tasks/Attach10xBarcodes.wdl

```diff
@@ -0,0 +1,31 @@
+
+task Attach10xBarcodes {
+  File r1
+  File i7
+  File u2
+
+  # estimate disk requirements
+  Float bam_size = size(u2, "G")
+  Float fastq_size = size(r1, "G") + size(i7, "G")
+  Int estimated_required_disk = ceil(fastq_size + bam_size * 2.2)
+  # other parameters
```

Comment:
There are no other parameters right now, so I think this comment can be deleted.

Reply:
👍